systemtests/test_env.py

```diff
@@ -89,56 +88,71 @@ def test_env(ctx, env):
             ctx.invoke(
                 upload_symbols,
                 expect_code=201,
-                auth_token=f"{auth_token}",
-                base_url=f"{tecken_host}",
+                auth_token=auth_token,
+                base_url=tecken_host,
                 symbolsfile=f"{ZIPS_DIR}{fn}",
             )
+        click.echo("")
 
         click.echo(
             click.style(">>> UPLOAD BY DOWNLOAD TEST (DESTRUCTIVE)", fg="yellow")
         )
-        URL = ctx.invoke(list_firefox_symbols_zips, number=1, max_size=1000000000)[0]
+        url = ctx.invoke(list_firefox_symbols_zips, number=1, max_size=1000000000)[0]
         ctx.invoke(
             upload_symbols_by_download,
-            base_url=f"{tecken_host}",
-            auth_token=f"{auth_token}",
-            url=f"{URL}",
+            base_url=tecken_host,
+            auth_token=auth_token,
+            url=url,
         )
-        click.echo("\n")
     else:
-        click.echo(click.style(">>> SKIPPING DESTRUCTIVE TESTS\n", fg="yellow"))
-
-    if bad_token_test == 1:
-        # This tests a situation that occurs when nginx is a reverse-proxy to
-        # tecken and doesn't work in the local dev environment. bug 1655944
+        click.echo(click.style(">>> SKIPPING DESTRUCTIVE TESTS", fg="yellow"))
+    click.echo("")
+
+    if bad_token_test:
+        # Pick a zip file that's > 120kb and upload it with a bad token. This tests a
+        # situation that occurs when nginx is a reverse-proxy to Tecken and doesn't
+        # correctly return a response if the stream isn't exhausted. This doesn't work
+        # in the local dev environment. bug #1655944
         click.echo(
             click.style(
-                ">>> UPLOAD WITH BAD TOKEN TEST--this should return a 403 and error and not a RemoteDisconnected",
+                (
+                    ">>> UPLOAD WITH BAD AUTH TOKEN TEST--this should return an HTTP "
+                    "403 and not a RemoteDisconnected"
+                ),
                 fg="yellow",
             )
         )
-        first_file = [
-            name
+        zip_files = [
+            f"{ZIPS_DIR}{name}"
             for name in os.listdir(f"{ZIPS_DIR}")
-            if os.path.isfile(f"{ZIPS_DIR}{name}")
-        ][0]
-        ctx.invoke(
-            upload_symbols,
-            expect_code=403,
-            auth_token="badtoken",
-            base_url=f"{tecken_host}",
-            symbolsfile=f"{ZIPS_DIR}{first_file}",
-        )
+            if (
+                os.path.isfile(f"{ZIPS_DIR}{name}")
+                and os.path.getsize(f"{ZIPS_DIR}{name}") > 120_000
+            )
+        ]
 
-    click.echo("\n")
+        if not zip_files:
+            click.echo(
+                click.style(
+                    ">>> No zip files > 120kb; skipping bad token test", fg="red"
+                )
+            )
```

Comment:
Do we want to make sure there's a zip file that's > 120kb as part of setup_tests.py, so that we don't risk not running this test?